client/gutenberg/extensions/presets/jetpack/index-beta.json

```diff
@@ -1,4 +1,5 @@
 [
   "related-posts",
+  "tiled-gallery",
```

Comment:
Let's drop this for now and add it when we're in a better state.

Reply:
Done https://github.com/Automattic/wp-calypso/pull/27458/commits/e3f65a73d1e86bc11c9683a205243477facacd1c